tests/hardhat/Fork/diamondTest.ts

```diff
@@ -0,0 +1,414 @@
+          expect(await diamondUnitroller.closeFactorMantissa()).to.equals(parseUnits(currentCloseFactor, 0));
+        });
+
+        it("setting collateral factor", async () => {
+          await diamondUnitroller.connect(owner)._setCollateralFactor(vUSDT.address, 2);
+          market = await diamondUnitroller.markets(vUSDT.address);
+          expect(market.collateralFactorMantissa).to.equal(2);
+
+          await diamondUnitroller.connect(owner)._setCollateralFactor(vUSDT.address, parseUnits("8", 17));
+          market = await diamondUnitroller.markets(vUSDT.address);
+          expect(market.collateralFactorMantissa).to.equal(parseUnits("8", 17));
+        });
+
+        it("setting setting Liquidation Incentive", async () => {
+          await diamondUnitroller.connect(owner)._setLiquidationIncentive(parseUnits("13", 17));
+          expect(await diamondUnitroller.liquidationIncentiveMantissa()).to.equal(parseUnits("13", 17));
+
+          await diamondUnitroller.connect(owner)._setLiquidationIncentive(parseUnits("11", 17));
+          expect(await diamondUnitroller.liquidationIncentiveMantissa()).to.equal(parseUnits("11", 17));
+        });
+
+        it("setting Pause Guardian", async () => {
+          const currentPauseGuardia = (await diamondUnitroller.pauseGuardian()).toString();
+
+          await diamondUnitroller.connect(owner)._setPauseGuardian(owner.address);
+          expect(await diamondUnitroller.pauseGuardian()).to.equal(owner.address);
+
+          await diamondUnitroller.connect(owner)._setPauseGuardian(currentPauseGuardia);
+          expect(await diamondUnitroller.pauseGuardian()).to.equal(currentPauseGuardia);
+        });
+
+        it("setting market borrow cap", async () => {
+          const currentBorrowCap = (await diamondUnitroller.borrowCaps(vUSDT.address)).toString();
+          await diamondUnitroller.connect(owner)._setMarketBorrowCaps([vUSDT.address], [parseUnits("10000", 18)]);
+          expect(await diamondUnitroller.borrowCaps(vUSDT.address)).to.equal(parseUnits("10000", 18));
+
+          await diamondUnitroller.connect(owner)._setMarketBorrowCaps([vUSDT.address], [currentBorrowCap]);
+          expect(await diamondUnitroller.borrowCaps(vUSDT.address)).to.equal(currentBorrowCap);
+        });
+
+        it("pausing mint action in vUSDT", async () => {
+          expect(await diamondUnitroller.connect(owner)._setActionsPaused([vBUSD.address], [0], true)).to.emit(
+            vBUSD,
+            "ActionPausedMarket",
+          );
+
+          await expect(vBUSD.connect(usdtHolder).mint(1000)).to.be.revertedWith("action is paused");
+
+          expect(await diamondUnitroller.connect(owner)._setActionsPaused([vBUSD.address], [0], false)).to.emit(
+            vBUSD,
+            "ActionPausedMarket",
+          );
+          expect(await vBUSD.connect(busdHolder).mint(10)).to.be.emit(vBUSD, "Mint");
+        });
```

Comment:
This test is failing. I think it's because now the function `_setActionsPaused` is checking for `_setActionsPaused(address[],uint8[],bool)`, and the original one was checking `_setActionsPaused(address[],uint256[],bool)`

Moreover, the description of the test mentions `vUSDT` but the test is done with `vBUSD`. `vBUSD` minting is currently paused. I would use `vUSDT`

Reply:
Used vUSDT